src/pages/industries/software.jsx

```diff
@@ -0,0 +1,219 @@
+  logo: ClickHouseLogo,
+  stats: [
+    {
+      heading: 'MASSIVE SCALE',
+      subHeading:
+        'Secured 10+PiB of streaming data & 30+ trillion inserted records in the first months of deployment',
+    },
+    {
+      heading: 'MULTI-CLOUD',
+      subHeading: '10,000+ pods across multiple regions and cloud providers',
+    },
+    {
+      heading: 'TIME TO VALUE',
+      subHeading: 'Weeks from POC to in-production deployment',
+    },
+  ],
+  url: 'https://www.cncf.io/case-studies/clickhouse/',
+  CTAtext: 'READ THE CASE STUDY',
+};
+
+const ascendStats = {
+  description:
+    'Ascend turned to Cilium as their CNI which simplified integrating into customer networks, eliminated their IP churn and density issues.',
+  logo: AscendLogo,
+  stats: [
+    {
+      heading: 'REDUCED IPS 256X',
+      subHeading: 'Running overlay giving pods non-VPC-routable IPs',
+    },
+    {
+      heading: 'COMPLETE DATA ENCRYPTION',
+      subHeading: 'At rest and in transit',
+    },
+    {
+      heading: 'FASTER DEBUGGING',
+      subHeading: 'From 4-16 hours down to 20 seconds',
+    },
+  ],
+  url: 'https://www.cncf.io/case-studies/ascend/',
+  CTAtext: 'READ THE CASE STUDY',
+};
+
+const bulletSection1 = {
+  heading: 'Align with DevOps  and  GitOps Principles',
+  paragraphs: [
+    "One of the core tenets of DevOps is automation. Cilium's architecture ensures that all its components can be deployed automatically, integrating into existing workflows and augmenting automation capabilities. Platform engineering teams can seamlessly integrate Cilium into their CI/CD pipelines, ensuring continuous delivery and integration without manual intervention.",
+    "Every aspect of  Cilium's Mesh can be configured using an API. This aligns with the GitOps principles, where the desired state of infrastructure is declared in code and stored in Git repositories. The API-driven model of Cilium enables organizations to manage, scale, and modify their infrastructure using code, ensuring consistency and repeatability.",
+  ],
+  withImage: true,
+  imageSrc: SoftwareImage1,
+  ImageAlt: 'cilium saas tools and dbs illustration',
+};
+
+const bulletSection2 = {
+  heading:
+    "Streamline Onboarding New Applications and Teams and Accelerate debugging with Cilium's Hubble Service Map",
+  paragraphs: [
+    "Cilium's Hubble Service Map feature allows platform teams to furnish application teams with a portal to view their own workloads, dependencies, and network flows. This self-service model makes it easier to identify service connections, calls, and even DNS operations, all crucial for diagnosing problems. The service map helps streamline the onboarding of new applications and teams and accelerates the debugging process.",
+    'By mitigating disputes between application and infrastructure operation teams, Cilium accelerates the investigation of application-layer issues, enabling frictionless, self-service root cause analysis for application teams.',
```

Comment:
```suggestion
    'By mitigating disputes between application and infrastructure operation teams, Cilium accelerates the investigation of application-layer issues, enabling frictionless, self-service root cause analysis for developers.',
```

Reply:
@xmulligan This is me being unnecessarily picky, but  I think the choice of wording here between "application teams" and "developers" might play a tad bit of a role in conveying the right message. This is consistent with other Cilium-related copies and documents I've looked at. This also emphasizes the separation between app and platform/operations teams and how Cilium empowers non-operation teams with tools to diagnose issues. 

Again, I don't trust my instinct here, but you get the point I'm trying to make.